### ISSUE TITLE
Improve Notification and Signup Panel's Responsiveness

### ISSUE THREAD

**Issue by brrkrmn** [unstructuredstudio/zubhub#991]
## Description

Notification and signup panels behave abnormally below a certain screen size. They should either be in their original place, or they can be full-screen, just like how we show notification panel to logged in mobile users. Waiting for suggestions to improve these, thanks. 

### To Reproduce:

1. Visit https://zubhub.unstructured.studio/
2. Make sure you are not logged in
3. Click on the notification or the avatar icons on the top right
4. Make your screen width below 600

![Screenshot 2023-11-07 at 09 57 12](https://github.com/unstructuredstudio/zubhub/assets/92817363/ce614bc1-ad94-496f-bdc5-9d8ee84aa66f)
![Screenshot 2023-11-07 at 09 57 42](https://github.com/unstructuredstudio/zubhub/assets/92817363/adf8c162-2010-4410-9e4c-b882708a2374)

**Comment by mehreeee** [comment]
Hi @brrkrmn is this a design issue?

**Comment by brrkrmn** [comment]
Hello @mehreeee yes it is open to design suggestions

**Comment by mehreeee** [comment]
Hey @brrkrmn this is how it looks on an iPhone 14 plus, width 428px. I don't think it should show a full screen
![Screenshot (85)](https://github.com/unstructuredstudio/zubhub/assets/105458062/53523cb7-e576-4f62-924e-9c3e09f51cd3)

![Screenshot (87)](https://github.com/unstructuredstudio/zubhub/assets/105458062/25c5ba0a-5983-436c-9abf-d2166e79f559)

**Comment by DonPresh** [comment]
@mehreeee leaning more into the second one. First one looks like it's too big for that screen size.

**Comment by mehreeee** [comment]
> @mehreeee leaning more into the second one. First one looks like it's too big for that screen size.

Thank you @DonPresh. @brrkrmn you can use the second size for it. The sizes should be the same on both modals to maintain consistency.

**Comment by DonPresh** [comment]
@mehreeee This is for mobile yeah? How about designing a different solution? Let me whip up something real quick. We can make further iterations to the design.

**Comment by DonPresh** [comment]
https://github.com/unstructuredstudio/zubhub/assets/100778015/a5d6ea43-80a5-47ff-b174-b9d24de771c7

@mehreeee and @brrkrmn what do you think?
A lil different from what we have (P.S, the texts are filler texts) but makes that place a little less cluttered.

**Comment by mehreeee** [comment]
Or we could do this @brrkrmn @DonPresh 
![Screenshot (88)](https://github.com/unstructuredstudio/zubhub/assets/105458062/5269f62c-8a88-431e-a154-40985b7ab70b)

**Comment by mehreeee** [comment]
> Screen.Recording.2023-11-07.at.12.35.18.mov 
> @mehreeee and @brrkrmn what do you think? A lil different from what we have (P.S, the texts are filler texts) but makes that place a little less cluttered.

Looks great. But they look more like links that will open in new tabs and will take a lot longer to implement. I'm assuming we want to keep users on the same page.

**Comment by aqsaaqeel** [comment]
The signup panel can just be centered on smaller screens I think @brrkrmn

**Comment by mehreeee** [comment]
Hi @brrkrmn the scrollbar issue has been fixed, good job. Like @NdibeRaymond suggested here https://github.com/unstructuredstudio/zubhub/pull/989#issuecomment-1793727487, we could also fix the same issue on the web. Like making the modal not cover the top nav bar here, but fix it directly below the notification bell.

![Screenshot (89)](https://github.com/unstructuredstudio/zubhub/assets/105458062/bfecb2f0-1aea-4985-ba7d-ed1ab78a0073)

**Comment by Dumzy006** [comment]
Nice Iterations @mehreeee @DonPresh 
I agree with @aqsaaqeel suggestion
I feel it would be better if  the notification and signup panels are  centered aligned reason been that : Center aligned modals adapts well to different screen sizes and orientations, and also draws the user's attention to the modal's content, making it the primary focal point.

See screenshot of what I designed below
![image](https://github.com/unstructuredstudio/zubhub/assets/136137889/a1744094-fd60-4f4c-a655-b9a59d265ee7)

**Comment by mehreeee** [comment]
> Nice Iterations @mehreeee @DonPresh I agree with @aqsaaqeel suggestion I feel it would be better if the notification and signup panels are centered aligned reason been that : Center aligned modals adapts well to different screen sizes and orientations, and also draws the user's attention to the modal's content, making it the primary focal point.
> 
> See screenshot of what I designed below ![image](https://user-images.githubusercontent.com/136137889/281129730-c82b370a-c3cb-46f0-9141-b70acc6995e5.png)

Nice design @Dumzy006. If it’s going to be centralized it might as well be a full screen because it’s kind of covering the other ‘share your work’ CTA button.

**Comment by Dumzy006** [comment]
Since you are logged out, the 'share your work' button still serves the same purpose as to that of the 'notification' and 'sign up' when clicked @meh 

> > Nice Iterations @mehreeee @DonPresh I agree with @aqsaaqeel suggestion I feel it would be better if the notification and signup panels are centered aligned reason been that : Center aligned modals adapts well to different screen sizes and orientations, and also draws the user's attention to the modal's content, making it the primary focal point.
> > See screenshot of what I designed below ![image](https://user-images.githubusercontent.com/136137889/281129730-c82b370a-c3cb-46f0-9141-b70acc6995e5.png)
> 
> Nice design @Dumzy006. If it’s going to be centralized it might as well be a full screen because it’s kind of covering the other ‘share your work’ CTA button.

Since you are logged out, the 'share your work' button still serves the same purpose as to that of the 'notification' and 'sign up' when clicked .... Any one works @mehreeee

**Comment by mehreeee** [comment]
> Since you are logged out, the 'share your work' button still serves the same purpose as to that of the 'notification' and 'sign up' when clicked @meh
> 
> > > Nice Iterations @mehreeee @DonPresh I agree with @aqsaaqeel suggestion I feel it would be better if the notification and signup panels are centered aligned reason been that : Center aligned modals adapts well to different screen sizes and orientations, and also draws the user's attention to the modal's content, making it the primary focal point.
> > > See screenshot of what I designed below ![image](https://user-images.githubusercontent.com/136137889/281129730-c82b370a-c3cb-46f0-9141-b70acc6995e5.png)
> > 
> > 
> > Nice design @Dumzy006. If it’s going to be centralized it might as well be a full screen because it’s kind of covering the other ‘share your work’ CTA button.
> 
> Since you are logged out, the 'share your work' button still serves the same purpose as to that of the 'notification' and 'sign up' when clicked @mehreeee

Yeah I agree. But The main CTA is the share your work. It shouldn’t be covered so users can chose any of the options to sign in. Just my opinion tho 👍

**Comment by brrkrmn** [comment]
@mehreeee, @DonPresh, @Dumzy006, @aqsaaqeel thank you guys for your great suggestions, I think centering the modal in the page looks really good, about it covering the 'share your work' button, both of the buttons redirect the user to the same page so I am not sure if it creates a problem

**Comment by brrkrmn** [comment]
> Hi @brrkrmn the scrollbar issue has been fixed, good job. Like @NdibeRaymond suggested here [#989 (comment)](https://github.com/unstructuredstudio/zubhub/pull/989#issuecomment-1793727487), we could also fix the same issue on the web. Like making the modal not cover the top nav bar here, but fix it directly below the notification bell.

@mehreeee thank you for reminding this, you're right this should also be fixed. We can also add the close buttons as you suggested as part of this issue since it looks like a small fix

**Comment by PiyushChandra17** [comment]
@srish @tuxology @brrkrmn I would like to work on this issue. Can you please assign this issue to me? Thanks :)
